Getting green ticks on the SPI build matrix

Hi, it's great to see this package on swift.org but I noticed that we're currently showing all grey boxes over on the [SPI compatibility matrix](https://swiftpackageindex.com/AparokshaUI/adwaita-swift):

![CleanShot 2024-03-26 at 11 17 21@2x](https://github.com/AparokshaUI/adwaita-swift/assets/65520/8c7810d6-0f7e-4e07-a944-b9d7f0d8ed18)

We currently don't support adding OS level dependencies for build on Apple platforms but our Linux builders are a bit more flexible. I've tried running a build locally with the dependencies set up but I'm getting build errors:

```
/host/adwaita-swift/Sources/Adwaita/Model/User Interface/Window/GTUIAboutWindow.swift:18:23: error: cannot find 'adw_about_window_new_from_appdata' in scope
            pointer = adw_about_window_new_from_appdata(filePath, nil)?.cast()
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/host/adwaita-swift/Sources/Adwaita/Model/User Interface/Window/GTUIAboutWindow.swift:18:67: error: 'nil' requires a contextual type
            pointer = adw_about_window_new_from_appdata(filePath, nil)?.cast()
                                                                  ^
```

Here are the steps to reproduce:

```
docker run --rm -it -v $PWD:/host -w /host --network="host" --entrypoint bash swift:5.10-jammy
apt-get update
apt-get install libadwaita-1-dev
git clone https://github.com/AparokshaUI/adwaita-swift
cd adwaita-swift
swift build
```

It looks like `libadwaita-1-dev` isn't the correct dependency. Do you know what dev package needs to be installed on Ubuntu?

--- a/.spi.yml
+++ b/.spi.yml
@@ -1,3 +1,17 @@
 version: 1
 external_links:
-    documentation: "aparokshaui.github.io/adwaita-swift/"
+  documentation: "aparokshaui.github.io/adwaita-swift/"
+builder:
+  configs:
+    - platform: linux
+      swift_version: '5.10'
+      image: registry.gitlab.com/finestructure/spi-images:adwaita-5.10-latest
+    - platform: linux
+      swift_version: '5.9'
+      image: registry.gitlab.com/finestructure/spi-images:adwaita-5.9-latest
+    - platform: linux
+      swift_version: '5.8'
+      image: registry.gitlab.com/finestructure/spi-images:adwaita-5.8-latest
+    - platform: linux
+      swift_version: '5.7'
+      image: registry.gitlab.com/finestructure/spi-images:adwaita-5.7-latest

